test/smf1-config-ref.yaml

```diff
@@ -1,15 +1,17 @@
 apiVersion: ref.nephio.org/v1alpha1
 kind: Config
+
 metadata:
-  name: smf-config-ref
   namespace: free5gc
+  name: smf1-config-ref
+
 spec:
   config:
     apiVersion: workload.nephio.org/v1alpha1
     kind: NFDeployment
     metadata:
-      name: free5gc-upf-1
-      namespace: upf-1
+      namespace: smf-1
```

Comment:
Hi in the old version it was "free5gc-upf-1" should we keep UPF here? I think smf config has reference of the upf?